src/layer/vulkan/pooling_vulkan.cpp

```diff
@@ -343,6 +396,40 @@ int Pooling_vulkan::forward(const VkMat& bottom_blob, VkMat& top_blob, VkCompute
         return 0;
     }
 
+    if (adaptive_pooling)
+    {
+        int outw = 1;
+        int outh = 1;
```

Comment:
hardcode outw outh ?
There are parameterized out_w and out_h

Reply:
fixed